grades/management/commands/check_final_grade_freeze_status.py

```diff
@@ -38,8 +38,9 @@ def handle(self, *args, **kwargs):  # pylint: disable=unused-argument
             )
         elif CourseRunGradingStatus.is_pending(run):
             group_results_id = CACHE_ID_BASE_STR.format(edx_course_key)
-            if cache_redis.get(group_results_id):
-                results = GroupResult.restore(group_results_id)
+            celery_results_id = cache_redis.get(group_results_id)
+            if celery_results_id:
```

Comment:
In `freeze_course_run_final_grades` it checks against `None` explicitly, so we should remove that check if that one needs to check against an empty string too